sdk/canton/BUILD.bazel

```diff
@@ -207,7 +207,10 @@ java_library(
 scala_macro_library(
     name = "community_lib_slick_slick-fork",
     srcs = glob(["community/lib/slick/src/main/**/*.scala"]),
-    scalacopts = ["-Wconf:cat=unused-imports:s"],
+    scalacopts = [
```

Comment:
:+1:

Reply:
I just think it was an oversight.

Reply:
maybe we could define a the top of the file a constant like 
```canton_scalacopts = ["-Xsource:3"]```
and use it in every canton rule, a bit like we are doing with `lf_scalacopt` for LF targets.